packages/apps/human-app/frontend/src/pages/worker/jobs/components/my-jobs/mobile/my-jobs-table-mobile.tsx

```diff
@@ -49,7 +49,8 @@ export function MyJobsTableMobile({
     hasNextPage,
   } = useInfiniteGetMyJobsData();
 
-  const { mutate: rejectTaskMutation } = useRejectTaskMutation();
+  const { mutate: rejectTaskMutation, isPending: isRejectPending } =
```

Comment:
@adrian-oleskiewicz 
This mutation is top-level and when you click "Reject" for some on mobile - all reject buttons share the same `isRejectPending` state. Could you please fix that as well so they are independent?

Reply:
@dnechay Good point. I refactored it a bit :) thanks